src/MerchantFrontend/src/app/views/merchant-login/merchant-login.component.ts

```diff
@@ -8,11 +11,17 @@ import {Router} from '@angular/router';
 })
 export class MerchantLoginComponent implements OnInit {
 
+  authorities: Authority[];
```

Comment:
why do we need authorities here?

Reply:
i think there is no router

Reply:
i removed these orphans from an old implementation.